js/read_receipts.js

```diff
@@ -4,14 +4,14 @@
 ;(function() {
     'use strict';
     window.Whisper = window.Whisper || {};
-    Whisper.ReadReceipts = new (Backbone.Collection.extend({
+    Whisper.ReadSyncs = new (Backbone.Collection.extend({
```

Comment:
It would make sense to rename the containing file here, right?

Reply:
Ah, I see. We've got `Whisper.ReadReceipts` defined below. We should probably these two ideas into separate files.